### ISSUE TITLE
[CoE Starter Kit - QUESTION] Power Platform Admin V2 for GCCH

### ISSUE THREAD

**Issue by yearwoodm15** [microsoft/coe-starter-kit#9129]
### Does this question already exist in our backlog?

- [x] I have checked and confirm this is a new question.

### What is your question?

Hello,

I am attempting to upgrade to the CoE Core Components 4.33, and within that solution I am unable to connect to Power Platform Admin V2 for GCCH with my service account. This error occurs with both my service account and admin account. My question to the team here is whether this is expected within the GCCH tenant? If so, when will this be fixed in GCCH and what does that mean for the constant updates we try to keep up with CoE?

Thank you!

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.33

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Hello. Unfortunately I am not able to assist with when this connector will be available for you. @Grant-Archibald-MS has a product issue on his plate which tracks this. Please see https://github.com/microsoft/coe-starter-kit/issues/8835 to follow for updates. 

However we did release a version of Sept 2024 (the latest build) which does not use this connector. 
Please see the note in https://github.com/microsoft/coe-starter-kit/releases/tag/CoEStarterKit-September2024
![Image](https://github.com/user-attachments/assets/8d827a42-0ce3-49ef-9acf-7f7cd36e7f07)

**Comment by Jenefer-Monroe** [comment]
I'm here to help!

**Comment by yearwoodm15** [comment]
Thanks Jenefer, I think we're on the right track now. Apologies as this is a bit of a different question, however I am getting an error when trying to use the CoE Setup Wizard with my service account. I have confirmed that the following are on:

![Image](https://github.com/user-attachments/assets/e910992e-0b77-497b-8a69-25e5e2fa312a)

**Comment by Jenefer-Monroe** [comment]
Having the first screen in the setup wizard be broken is kind of common, so I have some canned advice. 
Please let me know if this resolves it for you. 

## Setup Wizard broken First Screen
This usually means that the following flows which are supposed to be on when you install / upgrade are not on:
- SetupWizard>CreateGroup
- SetupWizard>GetCurrentEnvironment
- SetupWizard>GetTenantID
- SetupWizard>GetUserDetails

And that typically means that the **_HTTP with Microsoft Entra ID (preauthorized)_** connection was not configured correctly. 
Sometimes the product will setup one of these with the wrong URL
Please find this connector and validate its pointing to the graph URL for your tenant. 
Here are the strings for the graph endpoints. It looks like yours would be https://graph.microsoft.us
[Microsoft Graph and Graph Explorer service root endpoints](https://learn.microsoft.com/graph/deployments#microsoft-graph-and-graph-explorer-service-root-endpoints)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/d17f395e-d935-4e76-b62f-e6ce444897af)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/13cbb030-aa19-4e73-bf2b-cc4b0b692418)

**Comment by yearwoodm15** [comment]
Thank you Jenefer, I will investigate and try this on Monday and report back. Appreciate all of your help!

**Comment by yearwoodm15** [comment]
Just kidding, I was able to check this really quickly. Looks like this is as expected:

![Image](https://github.com/user-attachments/assets/fef7a549-2805-4024-b66d-f28aab9fc14c)

![Image](https://github.com/user-attachments/assets/2bf7372d-2bf0-4782-b477-76b4c6bc5832)

**Comment by Jenefer-Monroe** [comment]
Can you please share what error you are seeing? Perhaps its a different one than I anticipated.

**Comment by yearwoodm15** [comment]
Sure thing, please see below:

![Image](https://github.com/user-attachments/assets/321080a8-39b0-4c0f-adae-697332a5281a)

**Comment by Jenefer-Monroe** [comment]
OK that is not the error I was thinking. We dont use a substring error anywhere in the app, I dont think its an expression in PowerApps

Can you see if you have unmanaged layers on that page? 
![Image](https://github.com/user-attachments/assets/841dc682-bc74-41b1-805e-6e93b985c51f)

And also share a screenshot like this so we can see what solutions you have installed in this environment? Be sure to show all records (all rows) and expand the indicated columns so that we can see full names
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/8dd512cc-21fc-4c64-adab-0f64c29b9bf2)

**Comment by yearwoodm15** [comment]
Hi Jenefer,

Please see below:

![Image](https://github.com/user-attachments/assets/59fe8d42-e433-4203-aee5-869785e09c03)
![Image](https://github.com/user-attachments/assets/16d39040-ad08-4365-882f-5884a432d37e)

**Comment by Jenefer-Monroe** [comment]
OK it must be something from the creator kit or the product as that is not a PowerApps function. 
Can you please hit download log file and share that? 
What happens if you hit ok?

**Comment by yearwoodm15** [comment]
Apologies, where would you like me to go? Is this within the solution?

**Comment by Jenefer-Monroe** [comment]
I dont know what happened to my comment, let me try again :)

This must be something with the creator kit (although I"m on the same build) or the product as that is not an expression for PowerApps.

- Can you please share the downloaded log file? 
- What happens if you hit "OK"?

**Comment by Jenefer-Monroe** [comment]
ok yes its a creator kit issue, one that I do not repro though I am on the same Creator Kit build.

(wrapped.https://pafeblobprodusgovhightx.blob.core.usgovcloudapi.net/alt20241016t000000zca697291863543bfbddcbff73ff282c1/20241016T183156Z/PCFControls/cat_PowerCAT.FluentDetailsList/PowerCAT.FluentDetailsList.bundle.js;:16:2279)
    at eval

- What region your envt is in as well as its type? This can be found in the product admin pages: https://admin.powerplatform.microsoft.com/
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/4c23e8c8-2f8b-4d8b-af43-79a9f289be69)

- If you step forward does the rest of the app work?

**Comment by yearwoodm15** [comment]
It is GCCH:

![Image](https://github.com/user-attachments/assets/2fc8d6ad-1625-43df-a107-30ba74fb02a1)

**Comment by Jenefer-Monroe** [comment]
That page shows you the below. Basically ensuring you have the correct licenses.

![Image](https://github.com/user-attachments/assets/0b53a56f-4c83-4b99-99c9-1be66e119506)

You may want to post a new issue with that error text in the Creator Kit bug list. In case they can do something for your case too. 
https://github.com/microsoft/powercat-creator-kit/issues

**Comment by yearwoodm15** [comment]
Thank you Jenefer, I have created [#501](https://github.com/microsoft/powercat-creator-kit/issues/501) for additional assistance.

To confirm from your previous response, am I okay to proceed with the Setup Wizard even with the error screen showing - or should I hold off until further assistance? My concern is whether there may be other unaware/backend errors when attempting to complete the Wizard.

**Comment by Jenefer-Monroe** [comment]
yes you can proceed. That page is not really important. And hopefully the rest of the screens will work correctly. Please do let me know.

**Comment by Jenefer-Monroe** [comment]
Did the rest of the pages work? I've been hoping so!

**Comment by yearwoodm15** [comment]
So sorry for the delayed response! Yes, I was able to proceed with the rest of the build as expected. I could not seem to perform the Dataflow (preview) piece of the Setup Wizard, it was grayed out. Is that currently expected in the GCCH tenant as well?

Thank you again for reaching out!

**Comment by Jenefer-Monroe** [comment]
Wonderful. The Dataflow piece is only if you are trying to setup with the old Data Export V1, an abandoned path that will not hit GA. 
So you are good to go with the Sync Flow side. 

Wonderful news!

**Comment by yearwoodm15** [comment]
Gotcha, makes sense. Sorry, one more question if you don't mind. I'm having trouble accessing App Usage stats in my Power BI dashboard, but I know it is from the environment variables. If Dataflow is not needed, how am I to retrieve this? Sorry if I am mixing this up, I am drawing a blank on how to get the "Dataflow ID" data:

![Image](https://github.com/user-attachments/assets/fe3422f9-2328-45e2-9b2b-08fc45f07f4c)

**Comment by Jenefer-Monroe** [comment]
No problem. App usage is done via the following today. Eventually the product's data export features will do this but today its only via the audit logs

[Collect audit logs using Microsoft Graph API](https://learn.microsoft.com/power-platform/guidance/coe/setup-auditlog-http-graphapi)

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
